exercises.py

```diff
@@ -0,0 +1,296 @@
+# Please answer the practice questions at the end of chapters 1-6 here under the cooresponding comment:
+
+# Chapter 1
+1. Which of the following are operators, and which are values?
+* = OPERATOR
+'hello' = STRING
+-88.8 = VALUE
+- = OPERATOR
+/ = OPERATOR
++ = OPERATOR
+5 = OPERATOR
+
+2. Which of the following is a variable, and which is a string?
+spam 
+'spam' = STRING
+
+3. Name three data types. = INTEGER, FLOAT, STRING
+
+4. What is an expression made up of? What do all expressions do? = EXPRESSION IS MADE UP OF OPERATORS AND VALUES. EXPRESSIONS DO SIMPLE MATH AND EVALUATE TO A SINGLE  VALUE.
+
+5. This chapter introduced assignment statements, like spam = 10. What is the difference between an expression and a statement? AN ASSIGNMENT STATEMENT STORES THE INTEGER VALUE.
```

Comment:
That's true, and you can assign any value to a variable (string, integer, None, etc).